cmd/relay/endpoint/name.go

```diff
@@ -1,15 +1,18 @@
 package endpoint
 
 import (
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/gin-gonic/gin"
+	cfg "github.com/iden3/go-iden3/cmd/relay/config"
 	common3 "github.com/iden3/go-iden3/common"
 	"github.com/iden3/go-iden3/services/namesrv"
 )
 
 func handleVinculateId(c *gin.Context) {
```

Comment:
shouldn't this handler be removed from here?

Reply:
done, now is only in nameserver, not more in relay